lib/jsdom/browser/file/blob.js

```diff
@@ -0,0 +1,78 @@
+"use strict";
+
+function Blob() {
+  if (!(this instanceof Blob)) {
+    throw new TypeError("DOM object constructor cannot be called as a function.");
+  }
+  var parts = arguments[0];
+  var properties = arguments[1];
+  if (arguments.length > 0) {
+    if (!parts || typeof parts !== "object" || parts instanceof Date || parts instanceof RegExp) {
+      throw new TypeError("blobParts should be nor a non-objects nor Dates nor RegExps nor null");
+    }
+  }
+
+  const buffers = [];
+
+  if (parts) {
+    var l = +parts.length;
+    for (var i = 0; i < l; i++) {
+      var part = parts[i];
+      var buffer;
+      if (part instanceof ArrayBuffer) {
+        buffer = new Buffer(new Uint8Array(part));
```

Comment:
OK. But Buffer is not part of the web, so for example it should not be supported as an argument to the Blob constructor. And if you don't use too many special Buffer features then I'd prefer Uint8Array, but I'll leave that decision up to you.